src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -337,12 +337,12 @@
     </ItemGroup>
   </Target>
 
-    <!-- Copy extra package ref content for our installers. '$(Pkg*)' props are set during NuGet restore when GeneratePathProperty="true" -->
+  <!-- Copy package ref symbols for our installers. '$(Pkg*)' props are set during NuGet restore when GeneratePathProperty="true" -->
   <Target Name="_CopyExtraPackageContent"
-      Inputs="$(PkgMono_Posix_NETStandard)\runtimes\osx\lib\netstandard2.0\Mono.Posix.NETStandard.dll;$(PkgXamarin_Build_AsyncTask)\lib\netstandard2.0\Xamarin.Build.AsyncTask.pdb"
-      Outputs="$(OutputPath)\Mono.Posix.NETStandard.dll;$(OutputPath)\Xamarin.Build.AsyncTask.pdb" >
+      Inputs="$(PkgXamarin_LibZipSharp)\lib\$(TargetFrameworkNETStandard)\libZipSharp.pdb;$(PkgXamarin_Build_AsyncTask)\lib\$(TargetFrameworkNETStandard)\Xamarin.Build.AsyncTask.pdb"
+      Outputs="$(OutputPath)\libZipSharp.pdb;$(OutputPath)\Xamarin.Build.AsyncTask.pdb" >
     <Copy
-        SourceFiles="$(PkgMono_Posix_NETStandard)\runtimes\osx\lib\netstandard2.0\Mono.Posix.NETStandard.dll;$(PkgXamarin_Build_AsyncTask)\lib\netstandard2.0\Xamarin.Build.AsyncTask.pdb"
+        SourceFiles="$(PkgXamarin_LibZipSharp)\lib\$(TargetFrameworkNETStandard)\libZipSharp.pdb;$(PkgXamarin_Build_AsyncTask)\lib\$(TargetFrameworkNETStandard)\Xamarin.Build.AsyncTask.pdb"
```

Comment:
Are these symbols not being copied to `$(OutputPath)` already? If `libZipSharp.dll` makes it there, I would think `libZipSharp.pdb` would be there, too?

I'm wondering if we can delete this entire target.

Reply:
Short form style projects do not copy NuGet assets by default, but we override this behavior with https://github.com/xamarin/xamarin-android/blob/master/Directory.Build.props#L12 

From what I've seen in local testing, this property doesn't extend to symbol files. There's a long discussion about the lack of symbol copy functionality in the new project system in https://github.com/dotnet/sdk/issues/1458. As far as I can tell such functionality still doesn't exist. I think we have a couple options for the NuGets we own that would allow us to remove this hack in the future:

1) Update the NuGets we own to generate symbols in a `.snupkg` file: https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages-snupkg.
2) Include symbols in the assembly? - https://github.com/dotnet/sdk/issues/1458#issuecomment-576961024